SDK Samples/C#/ASP.NET Core/BoldBI.Embedded.Sample/Views/Home/Index.cshtml

```diff
@@ -6,14 +6,9 @@
 
 <!DOCTYPE html>
 <html>
-
 <head>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700" />
     <link rel="stylesheet" href="~/css/Site.css" />
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jsrender/1.0.5/jsrender.min.js"></script>
-    <script type="text/javascript" src="https://cdn.boldbi.com/embedded-sdk/v5.3.53/boldbi-embed.js"></script>
+    <script type="text/javascript" src="https://cdn.boldbi.com/embedded-sdk/v6.2.7/boldbi-embed.js"></script>
```

Comment:
Set the CDN version as latest, it would be work for latest server inputs.

Reply:
CDN version changed for latest.